src/org/jitsi/impl/neomedia/transform/dtls/DtlsControlImpl.java

```diff
@@ -518,14 +643,14 @@ public DtlsControlImpl(boolean srtpDisabled)
 
         // The methods generateKeyPair(), generateX509Certificate(),
         // findHashFunction(), and/or computeFingerprint() may be too CPU
-        // intensive to invoke for each new DtlsControlImpl instance. That's why
-        // we've decided to reuse their return values within a certain time
+        // intensive to invoke for each new DtlsControlImpl instance. That's
+        // why we've decided to reuse their return values within a certain time
         // frame. Attempt to retrieve from the cache.
         synchronized (DtlsControlImpl.class)
         {
             certificateInfo = certificateInfoCache;
             if (certificateInfo == null
-                    || certificateInfo.timestamp + ONE_DAY
+                    || certificateInfo.timestamp + CERT_CACHE_EXPIRE_TIME
```

Comment:
In my view, the certificate cache expiration and the certificate expiration are loosely related but really are different concepts.
What if I want the certificate to expire after one day, but I want the cache to expire after 1 minute?